app/views/pages/index.html.erb

```diff
@@ -1,4 +1,10 @@
 <div class="container text-center">
   <h1>Pages#index</h1>
   <p>Find me in app/views/pages/index.html.erb</p>
+  <% if logged_in? %>
```

Comment:
Sort of covered in the comment above. This method is really just the same as `current_user`  its slightly more semantic to be able to use `logged_in?` but not by a lot.

Consider if you need the `logged_in?` method at all.